styles/auth.css

```diff
@@ -0,0 +1,20 @@
+.auth-container {
+  background-color: var(--bg-offgrey);
+  border-radius: 10px;
+  max-width: 30rem;
+  margin: 5rem auto;
+  padding: 1rem;
+}
+
+.user-details {
+  padding: 1rem;
+}
+.user-details > * {
+  margin: 0.5rem;
+}
+.login-preference {
+  margin-left: auto;
+}
+.user-details a:hover {
+  text-decoration: underline;
```

Comment:
suggestion:for general properties utility classes can be made so you dont have to repeat your code